object/user.go

```diff
@@ -183,6 +183,7 @@ type User struct {
 	MfaPhoneEnabled     bool                  `json:"mfaPhoneEnabled"`
 	MfaEmailEnabled     bool                  `json:"mfaEmailEnabled"`
 	MultiFactorAuths    []*MfaProps           `xorm:"-" json:"multiFactorAuths,omitempty"`
```

Comment:
Add `Invitation string `xorm:"varchar(100) index" json:"invitation"`

Also record the invitation's name in User object